File: projects/m6/001-lotto-game/solutions/nicoladurante/model/bill.js
export class Bill {
  amountOfNumbers;
  bet;
  wheel;

  constructor(amount, bet, wheel) {
    this.amountOfNumbers = amount;
    this.bet = bet;
    this.wheel = wheel;
  }

  static getAmountOfNumbers = (amount) => {
    return amount >= 1 && amount <= 10 ? Number(amount) : null;
  };
}

File: projects/m6/003-lotto-calculate-prizes/solutions/federicosiddi/lotto/bill.py
class Bill:
    """
    Class that defines a Bill object
    """
    def __init__(self, city, b_type, nums, bill_num):
        """
        Constructor function of the bill object

        :param city: type str, a valid city name, see Cities class
        :param b_type: type int, a valid bet type, see BetTypes class
        :param nums: type list, a list of random generated integer numbers from 1 to 90
        :param bill_num: type int, the actual number of the bill object
        """
        self.city = city
        self.type = b_type
        self.numbers = nums
        self.bill_num = bill_num
